### ISSUE TITLE
add aggregation tests which involve unbound variables

### ISSUE THREAD

**Issue by lisp** [w3c/rdf-tests#39]
two tests have been [proposed](https://github.com/kasei/rdf-tests/commit/389617a278b737ef6d61af12dfb94f9175923cc0) to demonstrate permitted aggregation results in the presence of unbound variables.

it would be worthwhile, in addition, to include tests which distinguish results where an implementation suppresses unbound values where permitted.

- one test which fails if the respective sample result is  unbound. this should be associated with a new feature, to document which of the alternative behaviours an implementation follows.
- one test which requires that an unbound value be valid as a group key. i did not see this yet covered, but may have overlooked a test which already does.

**Comment by gkellogg** [comment]
It's probably useful to surface this on public-rdf-tests@w3.org and public-sparql-dev@w3.org, as not everyone interested in this subscribes to the GitHub issues feed.

**Comment by gkellogg** [comment]
:8ball: As I see has been done: https://lists.w3.org/Archives/Public/p…

**Comment by gkellogg** [comment]
A PR with proposed tests and results would be welcome.

**Comment by lisp** [comment]
yes, this is a follow-on to that post (note, something seems broken in the lists url above)

before proposing tests, it would be good to have some degree of consensus concerning the content:

- whould it be appropriate to a feature?
- if so, what it might be named? would it be 'mf' or 'sd'?
-  should it be reflected in the service description?

**Comment by joernhees** [comment]
just linking this more directly: https://github.com/kasei/rdf-tests/commit/389617a278b737ef6d61af12dfb94f9175923cc0 @kasei

**Comment by gkellogg** [comment]
@lisp, perhaps a conversation referencing this thread, on public-rdf-test@w3.org and public-sparql-dev@w3.org might spur more discussion.
